ui/app/mainui/AppMain.qml

```diff
@@ -1328,38 +1328,32 @@ Item {
             id: sendModal
             active: false
 
-            function open(address = "") {
+            function open() {
                 this.active = true
-                this.item.addressText = address;
                 this.item.open()
             }
             function closed() {
                 // this.sourceComponent = undefined // kill an opened instance
                 this.active = false
             }
-            property var selectedAccount
             property var preSelectedHolding
             property string preSelectedHoldingID
             property int preSelectedHoldingType
-            property int sendType: -1
+            property int preSelectedSendType: -1
```

Comment:
@Khushboo-dev-cpp why don't you use `Constants.SendType.Unknown` instead of `-1` here?